.buildkite/pipeline.ml.yml

```diff
@@ -472,7 +472,7 @@
     # Dask tests and examples.
     - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=-client python/ray/util/dask/...
 
-- label: "Dataset tests"
+- label: "Dataset tests (Arrow - Latest Release)"
```

Comment:
Since Arrow is extremely fundamental to Datasets and since there are a lot of breaking changes between Arrow versions, I think that it's warranted. We need to support Arrow 6 until we drop Python 3.6, and as you know, we have users that are stuck on Python 3.6 for the foreseeable future that we want to ensure are successful. Supporting Arrow 7 and 8 is minimal effort on top of Arrow 9, so I'm proposing that we start with full coverage and then work on a deprecation plan that we can communicate to users.